src/main/java/org/broadinstitute/hellbender/engine/spark/AuthHolder.java

```diff
@@ -0,0 +1,141 @@
+package org.broadinstitute.hellbender.engine.spark;
+
+import com.google.api.services.genomics.Genomics;
+import com.google.api.services.storage.Storage;
+import com.google.cloud.dataflow.sdk.options.PipelineOptionsFactory;
+import com.google.cloud.genomics.dataflow.utils.GCSOptions;
+import com.google.cloud.genomics.utils.GenomicsFactory;
+
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.exceptions.UserException;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.io.Serializable;
+import java.security.GeneralSecurityException;
+
+/**
+ * Contains the authentication information for this application. This is used when accessing Google services such as GCS
+ * or Google Genomics.
+ *
+ * Conveniently, makeStorageClient() gives you authenticated access to GCS and makeGenomicsService() to Google Genomics.
+ */
+public class AuthHolder implements Serializable {
```

Comment:
Should this be called something like `GCPAuthHolder`, or is the intention for this to eventually store non-Google-Cloud-Platform auth info?